utils/monitoring/init-rum.ts

```diff
@@ -0,0 +1,20 @@
+import { AgentConfigOptions, init, apm } from '@elastic/apm-rum'
+
+const serviceName = process.env.APP || 'pa-front-local'
+const config: AgentConfigOptions = {
+  serviceName: `rum-${serviceName}`,
+  serverUrl: process.env.APM_URL || '',
+  environment: process.env.ENVIRONMENT || 'development',
+  active: process.env.APM_IS_ACTIVE === 'true',
+  distributedTracingOrigins: [process.env.API_ENDPOINT || ''],
+}
+
+export function initRum() {
+  return init(config)
+}
+
+export function captureRumError(error: Error | string) {
```

Comment:
nitpick:
```suggestion
export function captureRUMError(error: Error | string) {
```
comme il s'agit d'un acronyme et selon doc